web/src/main/java/org/wso2/testgrid/web/api/TestPlanService.java

```diff
@@ -107,4 +113,32 @@ public Response getTestPlan(@PathParam("id") String id, @QueryParam("require-tes
                     new ErrorResponse.ErrorResponseBuilder().setMessage(msg).build()).build();
         }
     }
+
+    /**
+     * This has the impleme
```

Comment:
Incomplete javadocs. Please fill and end it with a full-stop. Full-stop is mandatory for checkstyle in Windows.

Reply:
Fixed.